test/Microsoft.NET.Publish.Tests/GivenThatWeWantToPublishAProjectWithDependencies.cs

```diff
@@ -195,6 +197,106 @@ public void It_publishes_projects_with_filter_and_rid()
 
 //TODO: Enable testing the run once dotnet host has the notion of looking up shared packages
         }
+
+
+        [Theory]
+        [MemberData(nameof(PublishDocumentationExpectations))]
```

Comment:
Why use member data here and inlinedata in the other? I particularly prefer inlinedata in this case.